### DISCUSSION TITLE
TV2 Sports

### DISCUSSION THREAD

**Discussion by MapGuy11** [iptv-org/community#681] [Q&A]
**Channel Name:** TV2 Sports
**Channel ID:** SEN502.us
**Country:** United States
**Language:** English
**Website:** https://www.tv2sports.com/
**Stream URL (optional):** http://musikfestlive.sectv.com/feed.html
**Notes (optional):** I highly doubt anyone can get a stream from this if you can you are an absolute rockstar! Everything is unencrypted HTTP traffic over the feed website so maybe someone can find something!

**Comment by Sphinxroot** [comment]
indeed it will be hard to get https://lookup.sectv.com/MyAccount/aspLogin.aspx package start at 23.99 per month

**Comment by MapGuy11** [comment]
I wonder since it's http we can find were the redirect goes to.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19687
